packages/calendar/Calendar.tsx

```diff
@@ -0,0 +1,211 @@
+import { h, Component, prop, emit } from 'skatejs';
+import styles from './Calendar.scss';
+
+import * as getYear from 'date-fns/get_year';
+import * as getMonth from 'date-fns/get_month';
+import * as addDays from 'date-fns/add_days';
+import * as getDate from 'date-fns/get_date';
+import * as startOfWeek from 'date-fns/start_of_week';
+import * as format from 'date-fns/format';
+import * as isToday from 'date-fns/is_today';
+import * as parse from 'date-fns/parse';
+import * as isEqual from 'date-fns/is_equal';
+import { css } from '../_helpers/css';
+import { Button } from '../button/Button';
+import { buildFormatLocale, LocaleType } from '../_helpers/buildFormatLocale';
+
+interface CalendarProps extends JSX.HTMLProps<HTMLElement | any> {
+  selectedDate?: string,
+  onDateChange?: Function,
```

Comment:
I dont know when we want to do it. Because `onDateChanged` in interface is for user which want to use component like `<Calendar onDateChange={this.dateChangeHandler}  />` where  **dateChangeHandler** is their function, not event.